src/main/java/lotto/model/LottoNumber.java

```diff
@@ -39,16 +36,26 @@ private static void validateLottoNumber(int lottoNumber) {
         }
     }
 
-    private static void validateNumeric(String lottoNumber) {
-        if (!lottoNumber.matches(NUMERIC_CHECK_REGEX)) {
+    private static int validateAndParseInt(String lottoNumber) {
+        int number;
+
+        try {
+            number = Integer.parseInt(lottoNumber);
+        } catch (Exception e) {
             throw new WrongNumberException();
         }
+
+        return number;
```

Comment:
```suggestion
        try {
            return Integer.parseInt(lottoNumber);
        } catch (Exception e) {
            throw new WrongNumberException();
        }
```
이렇게 표현하는 것과 동일하지 않을까요?